File: LeptonicaSharp/Functions/libversions.vb
Imports System.Runtime.InteropServices
Imports LeptonicaSharp.Enumerations
Partial Public Class _AllFunctions


' SRC\libversions.c (101, 1)
' getImagelibVersions() Return string of version numbers; e.g., libgif 5.0.3 libjpeg 8b (libjpeg-turbo 1.3.0) libpng 1.4.3 libtiff 3.9.5 zlib 1.2.5 libwebp 0.3.0 libopenjp2 2.1.0 Notes (1) The caller must free the memory.
' getImagelibVersions() as char *
'''  <summary>
''' 
'''  </summary>
'''  <remarks>
'''  </remarks>
'''   <returns></returns>
Public Shared Function getImagelibVersions() as String



	Dim _Result as String = LeptonicaSharp.Natives.getImagelibVersions( )

	Return _Result
End Function

End Class

File: CS_LIB_ALPHA/Functions/libversions.cs
using System;
using Enumerations;
using System.Collections.Generic;
using System.Runtime.InteropServices;

namespace LeptonicaSharp{
public partial class _All {

// Comments.vb - System.String GenerateFunctionComment(NClang_Test.IntFunction, NClang_Test.TargetLang, System.String[]) :96
// libversions.c (101, 1)
// getImagelibVersions() as String
// getImagelibVersions() as char *
///  <remarks>
///  </remarks>
///  <include file="..\CHM_Help\IncludeComments.xml" path="Comments/getImagelibVersions/*"/>
///   <returns></returns>

public static String getImagelibVersions(){



	String _Result = Natives.getImagelibVersions();
	


//  



	return _Result;
}


}
}
